### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by Yarrik**

### CODE REVIEW

File: OYarotski/01.HTML/task_02.Python_1.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Python 01</TITLE>
+    </HEAD>
+    <!--start-->
+    <BODY>
+        <h1>HOW-TO: Программа на Python, Часть 1</h1>
+        <ul>
+            <li>Программа на Python — часть 1</li>
```
Comment (pluhin) [inline OYarotski/01.HTML/task_02.Python_1.html]: This list should be italic

File: OYarotski/01.HTML/task_02.Python_2.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<pre>print len(months)</pre>
+        <p>которая вернёт 12.</p>
+        <p>Ещё один пример списков — категории в поваренной книге. Например...</p>
+<pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+        <p>Таким образом, в <strong>categories[0]</strong> хранится <i>'Main dish'</i>, а в <strong>categories[4]</strong> — <i>'Cookies'</i>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+        <hr>
+        <h2>Комментарии</h2>
+        <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+        <p>Это комментарий</p>
```
Comment (pluhin) [inline OYarotski/01.HTML/task_02.Python_2.html:40]: And where is comment here? :)

**Review by pluhin — CHANGES_REQUESTED** [review]
See comments, need work

**Comment by Yarrik** [comment]
please check again .

**Review by pluhin — APPROVED** [review]

**Pull request merged**